PovioKit/Classes/Utilities/Throttler/README.md

```diff
@@ -0,0 +1,23 @@
+#  Throttler
+
+Job manager executing only latest scheduled job after set delay.
+
+## Example
+
+We used `Throttler` for search functionality, where we delayed requesting search results. Executing a request each key press would be quite an overhead, and the throttler comes to the rescue.
+
+Usage example:
```

Comment:
We don't need this line `Usage example:` since the chapter already says `example`.